internal/gatewayapi/testdata/gateway-with-listener-with-tcproute-with-multiple-rules.out.yaml

```diff
@@ -76,16 +76,11 @@ tcpRoutes:
   status:
     parents:
     - conditions:
-      - lastTransitionTime: null
-        message: Route is accepted
-        reason: Accepted
-        status: "True"
-        type: Accepted
       - lastTransitionTime: null
         message: One and only one rule is supported
         reason: InvalidRule
         status: "False"
-        type: ResolvedRefs
+        type: Accepted
```

Comment:
The condition for multiple rules should be `Accepted` instead of `ResolvedRefs`.